### DISCUSSION TITLE
AE中正常，导出后在个别帧会闪烁，请解？

### DISCUSSION THREAD

**Discussion by zishu** [Tencent/libpag#2051] [Q&A] [answered]
**AE文件中核心就两部分：**
1、一个晃动手机的视频；
2、一张图追踪手机晃动；

**问题：**
导出PAG文件后，视频显示的是正常的，但是上边追踪的图会一闪一闪的；
（正确的不应该是一闪一闪的）

**附件：**
1、AE源文件
2、导出的PAG文件

请解，感谢；
[pag.zip](https://github.com/Tencent/libpag/files/13809482/pag.zip)

**Comment by zishu** [comment, marked as answer]
已解

**Reply by zhalice2011** [reply]
你好, 请问你是如何解决的
